Update EOS examples for the new api on the Jungle 2.0 testnet (#2)
oraclize/eos-examples: wolfram-alpha/README.md

# Oraclize's Wolfram Alpha Example

This repo is to demonstrate how you would work with the Oraclize **WolframAlpha** datasource.

> The `wolframrand` example allows you to *retrive a random number from 1 to 6; and, if the number is

Reviewer: ```suggestion
The `wolframrand.cpp` example allows you to retrieve a random number from 1 to 6; and, if the number is
```

> not 6, the callback launches a new query*.

Reviewer: ```suggestion
not 6, the `callback` launches a new query.
```

## Notes

> Oraclize replies to your `oraclize_query` with the *random number between 1 and 6 result* calling your `callback(...)`.

Reviewer: ```suggestion
Oraclize replies to your `oraclize_query` by calling your `callback(...)` with the *random number between 1 and 6 result*.
```

> You can search your transaction ID in one of the following links to verify it:

Reviewer: ```suggestion
You can search for your transaction ID in one of the following links to verify it:
```

> * :mag_right::ledger: [Blocks.io](https://jungle.bloks.io/): a block explorer for the Jungle 2.0 testnet.

Reviewer: ```suggestion
* :mag_right::ledger: [Blocks.io](https://jungle.bloks.io/): A block explorer for the Jungle 2.0 testnet.
```

> * :palm_tree::lion::palm_tree: [Jungle 2.0](https://monitor.jungletestnet.io/#home): a transaction explorer is available selecting *Get TX* on the Jungle 2.0 testnet website.

Reviewer: ```suggestion
* :palm_tree::lion::palm_tree: [Jungle 2.0](https://monitor.jungletestnet.io/#home): A transaction explorer is available by selecting *Get TX* on the Jungle 2.0 testnet website.
```
